database/users/db.go

```diff
@@ -24,6 +24,7 @@ const (
          						                	WHERE plugin_id = '86DC3AE7-9F3D-42CB-85A3-A71ABC3C3CB8'
          						                    AND attribute_name = 'wallet'
          						                    AND value->'wallet' ? $1);`
+	getUserByNameQuery          = `SELECT * FROM "user" WHERE profile->'name' ? $1;`
```

Comment:
maybe it's better to use `WHERE profile->>'name' = $1`?
I don't know the exact difference, but let's check? :)

Reply:
also I think it's better to write additional query only for existence testing, without getting the whole user data

Reply:
and the main question - what about decentralization?